solidity/contracts/token/libs/TokenRouter.sol

```diff
@@ -78,14 +78,15 @@ abstract contract TokenRouter is GasRouter, ITokenBridge {
     /**
      * @inheritdoc ITokenFee
      * @notice Implements the standardized fee quoting interface for token transfers based on
-     * overridable internal functions of _quoteGasPayment, _feeRecipientAmount, and
-     * _externalFeeAmount. The implementation assumes that feeRecipient and external
-     * fees are charged in the same token as the transfer. Currently no children override
-     * quoteTransferRemote itself.
+     * overridable internal functions of _quoteGasPayment, _feeRecipientAmount, and _externalFeeAmount.
      * @param _destination The identifier of the destination chain.
      * @param _recipient The address of the recipient on the destination chain.
      * @param _amount The amount or identifier of tokens to be sent to the remote recipient
      * @return quotes An array of Quote structs representing the fees in different tokens.
+     * @dev This function may return multiple quotes with the same denomination. Convention is to return:
+     *  - the native token first
```

Comment:
Express here that the first piece is explicitly the quoteDispatch fees like IGPs. Also this will not include the native amount of this is HypNative yes? I'd like to be a bit more precise here. 

Also maybe indicate that folks can use Quotes.extract, but also how they should generally quote offchain